target/linux/ramips/dts/mt7621_zte_mf286c3.dts

```diff
@@ -0,0 +1,316 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "zte,mf286c3", "mediatek,mt7621-soc";
+	model = "ZTE MF286C3";
+
+	chosen {
+		bootargs-override = "console=ttyS0,57600n8";
+	};
+
+	aliases {
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-names = "default";
+		pinctrl-0 = <&led_pins>;
+
+		led_status: led-0 { /* p0_6 - power_06 */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_STATUS;
+			gpios = <&aw9523 6 GPIO_ACTIVE_LOW>;
+		};
+
+		led-1 { /* p1_2 - wifi12 */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_WLAN;
+			gpios = <&aw9523 10 GPIO_ACTIVE_LOW>;
+		};
+
+		led-2 { /* p1_4 - phone14 */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_INDICATOR;
+			function-enumerator = <1>;
+			gpios = <&aw9523 12 GPIO_ACTIVE_LOW>;
+			default-state = "off";
```

Comment:
All `default-state = "off";` should be removed as this is the Linux default behavior.